packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
```

Comment:
You should add the columns like I did in this PR:
https://github.com/OfficeDev/office-ui-fabric-react/pull/5384/files
That way it'll render actual content =)

Reply:
I think using the same mechanism as DetailList.test.tsx is fine to create those snaps...

Reply:
Updated!